backend/app/services/resource_library/discovery.py

```
@@ -0,0 +1,340 @@
+                or DEFAULT_DISCOVERY_ASSISTANT_TEAM["namespace"],
+            },
+        }
+
+    def find_discovery_config_kind(self, db: Session) -> Kind | None:
+        return (
+            db.query(Kind)
+            .filter(
+                Kind.kind == DISCOVERY_CONFIG_KIND,
+                Kind.name == DISCOVERY_CONFIG_NAME,
+                Kind.namespace == DISCOVERY_CONFIG_NAMESPACE,
+                Kind.is_active == True,
```

Ruff (0.15.15)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)